runtime/auto-split.sh

```diff
@@ -7,7 +7,7 @@ n_outputs="$#"
 
 # Set a default DISH_TOP in this directory if it doesn't exist
 PASH_TOP=${PASH_TOP:-$(git rev-parse --show-toplevel)}
-
+# TODO change all auto-split calls to provide $distro as argument
```

Comment:
This TODO is no longer necessary here (also the distro finding) since we don't have a case expression below